### ISSUE TITLE
Column onupdate problem

### ISSUE THREAD

**Issue by Aya-Yang** [sqlalchemy/sqlalchemy#5155]
I have a column named updated_at, which is Datetime, and have onupdate. My problem is that I want to update another column without trigger this column's onupdate, how can I do it

**Comment by zzzeek** [comment]
if it's just a short thing you need to do once I might just use text, conn.execute("update table set foo=bar").   or make a new table() object that just has the columns you need and then use table.update().

The other way would be if you use table.update(), you would set the value of the "updated_at" column to be exactly the datetime() value that it already is, like if you SELECTed the row first.     the "onupdate" happens only if you don't provide a value for the column.

Basically there's no "simple" way for the onupdate to not trigger, you have to either provide a value or execute against something that doesn't have the onupdate.
